src/components/DappBrowser/control-panel/ControlPanel.tsx

```diff
@@ -158,7 +158,7 @@ export const ControlPanel = () => {
               wallet.type === WalletTypes.readOnly
                 ? i18n.t(i18n.l.wallet.change_wallet.watching)
                 : account.balance
-                  ? convertAmountToNativeDisplay(account.balance, nativeCurrency)
+                  ? account.balance
```

Comment:
`account.balance` is now the formatted version of the account balance - this will cause an issue on line 169 where it tries to convert it to a number; this will also cause issues in the sorting